Builds/Test.py

```diff
@@ -129,14 +169,76 @@ def powerset(iterable):
     help='Reduce output where possible (unit tests)',
 )
 
+# Scons and CMake parameters are too different to run
+# both side-by-side
+pgroup = parser.add_mutually_exclusive_group()
+
+pgroup.add_argument(
+    '--cmake', '-m',
```

Comment:
That's a good point. It seems easier to do without them now and add them in later than to shoehorn them in now and need to remove them later.